velox/docs/functions/spark/math.rst

```diff
@@ -80,6 +80,16 @@ Mathematical Functions
         SELECT 2L / 2L; -- 1.0
         SELECT 3 / 0; -- NULL
 
+.. spark:function:: long_divide(x, y) -> long
+
+    Returns the results of dividing x by y.
+    Supported type is LONG.
+    Overflow results return null. Corresponds to Spark's operator ``div``. ::
```

Comment:
Suggestion:
Returns NULL if ``y`` is zero or any input is NULL.

This can be removed if ``div`` is used as the function name:
"Corresponds to Spark's operator ``div``."